packages/cli-polaris/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@shopify/cli-polaris",
+  "version": "0.0.0",
+  "private": false,
+  "description": "Commands for managing Polaris in Shopify apps",
+  "bugs": {
+    "url": "https://github.com/Shopify/cli/issues"
+  },
+  "homepage": "https://github.com/shopify/cli#readme",
+  "type": "module",
+  "files": [
+    "/dist"
+  ],
+  "publishConfig": {
+    "access": "public",
+    "@shopify:registry": "https://registry.npmjs.org"
+  },
+  "scripts": {
+    "clean": "rimraf dist/",
+    "build": "tsc -b ./tsconfig.build.json",
+    "prepack": "cross-env NODE_ENV=production yarn run build && cp ../../README.md README.md",
+    "lint": "eslint \"src/**/*.ts\"",
+    "lint:fix": "eslint 'src/**/*.ts' --fix",
+    "test": "vitest run",
+    "test:watch": "vitest watch",
+    "tsc": "tsc --noEmit"
+  },
+  "eslintConfig": {
+    "extends": [
+      "../../.eslintrc.cjs"
+    ]
+  },
+  "dependencies": {
+    "@oclif/core": "1.9.2",
+    "@shopify/cli-kit": "3.8.0",
+    "@shopify/polaris-migrator": "^0.0.0",
```

Comment:
Does the tool already live in another package?

Reply:
Yes, the tool was build and then suggested that it be usable from the Shopify CLI. We could move all of the source code over to the CLI but thought this might be a good first-attempt strategy.